pkg/controller/drain.go

```diff
@@ -331,15 +338,7 @@ func (o *DrainOptions) deleteOrEvictPods(pods []api.Pod) error {
 		return o.client.Core().Pods(namespace).Get(name, metav1.GetOptions{})
 	}
 
-	if len(policyGroupVersion) > 0 {
-		err := o.evictPods(pods, policyGroupVersion, getPodFn, stopCh)
-		if err != nil {
-			glog.Warningf("Pod eviction was timed out, Error: %v. \nHowever, drain will continue to forcefully delete the pods by setting graceful termination period to 0s", err)
-		} else {
-			return nil
-		}
-	}
-	return o.deletePods(pods, getPodFn, stopCh)
+	return o.evictPods(len(policyGroupVersion) > 0, pods, policyGroupVersion, getPodFn)
```

Comment:
Should we change the function name now that this takes care of evicting and deleting pods?
Same for all the subsequent functions names.